README.rst

```diff
@@ -0,0 +1,502 @@
+enter the empty build directory and run::
+
+    $ ccmake /path/to/kwiver/source
+
+where the path above is the location of your kwiver source tree.  The
+ccmake tool allows for interactive selection of CMake options.
+Alternatively, using the CMake GUI you can set the source and build
+directories accordingly and press the "Configure" button.  When
+building with Fletch it is preferable to set the ``fletch_DIR`` on the
+command line like this::
+
+    $ ccmake /path/to/kwiver/source -Dfletch_DIR=/path/to/fletch/install
+
+Other CMake options can also be passed on the command line in this way
+if desired.
+
+CMake Options
+-------------
+
+The following are the most important CMake configuration options for KWIVER.
+
+* CMAKE_BUILD_TYPE -- The compiler mode, usually Debug or Release
+* CMAKE_INSTALL_PREFIX -- The path to where you want the kwiver build products to install
+* KWIVER_ENABLE_ARROWS -- Enable algorithm implementation plugins
+* KWIVER_ENABLE_C_BINDINGS -- Whether to build the Vital C bindings
+* KWIVER_ENABLE_DOCS -- Turn on building the Doxygen documentation
+* KWIVER_ENABLE_LOG4CPLUS -- Enable log4cplus logger back end
+* KWIVER_ENABLE_PYTHON -- Enable the Vital Python bindings (requires KWIVER_ENABLE_C_BINDINGS)
+* KWIVER_ENABLE_SPROKIT -- Enable the Stream Processing Toolkit
+* KWIVER_ENABLE_TESTS -- Build the unit tests
+* KWIVER_ENABLE_TOOLS -- Build the command line tools (e.g. plugin_explorer)
+* fletch_DIR -- Install directory for the Fletch support packages.
+
+There are many more options.  Specifically, there are numerous options
+for third-party projects prefixed with ``KWIVER_ENABLE_`` that enable
+building the Arrows plugins that depend on those projects.  When building
+with the support of Fletch_ (set ``fletch_DIR``) the enable options for
+packages built by Fletch should be turned on by default.
+
+
+Dependencies
+------------
+
+Vital has minimal required dependencies (only Eigen_).
+Sprokit additionally relies on Boost_.
+Arrows and Sprokit processes are structured so that
+the code that depends on an external package is in a directory with
+the major dependency name (e.g. vxl, ocv). The dependencies can be
+turned ON or OFF through CMake variables.
+
+.. _Boost: http://www.boost.org/
+
+Compiling
+---------
+
+Once your CMake command has completed and generated the build files,
+compile in the standard way for your build environment.  On Linux
+this is typically running ``make``.
+
+
+Running KWIVER
+==============
+
+Once you've built KWIVER, you'll want to test that it's working on
+your system.  From a command prompt execute the following command::
+
+	source </path/to/kwiver/build>/install/setup_KWIVER.sh
+
+Where `</path/to/kwiver/build>` is the actual path of your KWIVER
+CMake build directory.
+
+This will set up your PATH, PYTHONPATH and other environment variables
+to allow KWIVER to work conveniently.
+
+The central component of KWIVER is vital which supplies basic data
```

Comment:
@linus-sherrill I think we should move the rest of this section (starting at this line) to a separate document in `doc/manuals/sprokit` that is called `getting-started.rst` or something similar.  In the remainder of this section I would like to see a much simplified "hello world" example of how to run plugin_explorer, mostly to confirm that everything was built and/or installed correctly.  It should not be specific to sprokit, but indicate how to list all arrows plugins and all sprokit plugins.